frontend/src/components/common/form/InputGroup.tsx

```diff
@@ -18,6 +18,8 @@ type OptionalAttributes = {
   label?: string;
   /** The underlying HTML element to use when rendering the FormControl */
   as?: React.ElementType;
+  /** optional help text to display below the FormControl */
+  helpText?: string;
```

Comment:
is this getting passed through to the underlying input?

Reply:
you got it - see my previous comment